process/sync/shardblock.go

```diff
@@ -154,7 +150,6 @@ func NewShardBootstrap(
 	boot.setRequestedHeaderHash(nil)
 	boot.setRequestedMiniBlocks(nil)
 
-	boot.headersNonces.RegisterHandler(boot.receivedHeaderNonce)
```

Comment:
should also remove boot.receivedHeaderNonce (check first if it is not called from production code - it shouldn't be the case)

Reply:
method was removed.